src/autocomplete/databases/clickhouse/tests/revoke/revoke.test.ts

```diff
@@ -0,0 +1,557 @@
+        RELOAD EMBEDDED DICTIONARIES,
+        REMOTE,
+        RENAME COLUMN,
+        RESTART REPLICA,
+        ROLE ADMIN,
+        S3,
+        SECURITY NONE,
+        SELECT(test_column1, test_column2),
+        SHOW,
+        SHOW DATABASES,
+        SHOW TABLES,
+        SHOW COLUMNS,
+        SHOW DICTIONARIES,
+        SHOW ACCESS,
+        SHOW_USERS,
+        SHOW_ROLES,
+        SHOW_ROW_POLICIES,
+        SHOW_QUOTAS,
+        SHOW_SETTINGS_PROFILES,
+        SHOW CREATE USER,
+        SHOW CREATE ROLE,
+        SHOW POLICIES,
+        SHOW CREATE ROW POLICY,
+        SHOW CREATE POLICY,
+        SHOW CREATE QUOTA,
+        SHOW PROFILES,
+        SHOW CREATE SETTINGS PROFILE,
+        SHOW CREATE PROFILE,
+        SHOW NAMED COLLECTIONS SECRETS,
+        SHOW NAMED COLLECTIONS,
+        SHUTDOWN,
+        SOURCES,
+        SQL SECURITY NONE,
+        SQLITE,
+        START DISTRIBUTED SENDS,
+        START FETCHES,
+        START MERGES,
+        START MOVES,
+        START REPLICATED SENDS,
+        START REPLICATION QUEUES,
+        START SENDS,
+        START TTL MERGES,
+        STOP DISTRIBUTED SENDS,
+        STOP FETCHES,
+        STOP MERGES,
+        STOP MOVES,
+        STOP REPLICATED SENDS,
+        STOP REPLICATION QUEUES,
+        STOP SENDS,
+        STOP TTL MERGES,
+        SYNC REPLICA,
+        SYSTEM,
+        SYSTEM DISTRIBUTED SENDS,
+        SYSTEM DROP CACHE,
+        SYSTEM DROP DNS CACHE,
+        SYSTEM DROP DNS,
+        SYSTEM DROP MARK CACHE,
+        SYSTEM DROP MARK,
+        SYSTEM DROP UNCOMPRESSED CACHE,
+        SYSTEM DROP UNCOMPRESSED,
+        SYSTEM FETCHES,
+        SYSTEM FLUSH DISTRIBUTED,
+        SYSTEM FLUSH LOGS,
+        SYSTEM FLUSH,
+        SYSTEM KILL,
+        SYSTEM MERGES,
+        SYSTEM MOVES,
+        SYSTEM RELOAD CONFIG,
+        SYSTEM RELOAD DICTIONARIES,
+        SYSTEM RELOAD DICTIONARY,
+        SYSTEM RELOAD EMBEDDED DICTIONARIES,
+        SYSTEM RELOAD,
+        SYSTEM REPLICATED SENDS,
+        SYSTEM REPLICATION QUEUES,
+        SYSTEM RESTART REPLICA,
+        SYSTEM SENDS,
+        SYSTEM SHUTDOWN,
+        SYSTEM START DISTRIBUTED SENDS,
+        SYSTEM START FETCHES,
+        SYSTEM START MERGES,
+        SYSTEM START MOVES,
+        SYSTEM START REPLICATED SENDS,
+        SYSTEM START REPLICATION QUEUES,
+        SYSTEM START SENDS,
+        SYSTEM START TTL MERGES,
+        SYSTEM STOP DISTRIBUTED SENDS,
+        SYSTEM STOP FETCHES,
+        SYSTEM STOP MERGES,
+        SYSTEM STOP MOVES,
+        SYSTEM STOP REPLICATED SENDS,
+        SYSTEM STOP REPLICATION QUEUES,
+        SYSTEM STOP SENDS,
+        SYSTEM STOP TTL MERGES,
+        SYSTEM SYNC REPLICA,
+        SYSTEM TTL MERGES,
+        TABLE ENGINE,
+        TRUNCATE,
+        UPDATE,
+        URL,
+        USAGE
+      ON *.*
+      FROM test_user1, test_user2, CURRENT_USER;
+    `);
+    expect(autocompleteResult.errors).toHaveLength(0);
+});
+
+test('should not report errors', () => {
```

Comment:
I'd give these tests different names, because if one of them fails it will be hard to understand which one